demo/full/styles/style.css

```diff
@@ -58,9 +58,6 @@ body {
   display: inline;
 }
 
-.choice-input select {
-}
-
 .choice-input.text-input {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Unexpected empty block ([css:S4658](https://sonarcloud.io/organizations/rx-player/rules#rule_key=css%3AS4658))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=rx-player&pullRequest=264&issues=AWTMwmjYi6fIhhPMqNRR&open=AWTMwmjYi6fIhhPMqNRR)

[ISSUE_KEY]: AWTMwmjYi6fIhhPMqNRR